lib/model.js

```diff
@@ -270,18 +270,6 @@ Model.prototype.save = function (options, fn) {
       resolve(_this, numAffected);
     });
   });
-
-  return _this.$__handleSave(options).
```

Comment:
This return is made unreachable by the return on line [223](https://github.com/ChristianMurphy/mongoose/blob/linting/lib/model.js#L223)